packages/utils/src/types.ts

```diff
@@ -1,3 +1,6 @@
+import { ComponentType } from 'react';
```

Comment:
Can we add react as a peer dependency for @onaio/utils please?

Reply:
I am actually borderline on whether to have this in the utils package.  One one hand we use React everywhere, on the other hand we are introducing React as a dependency.  Thoughts at @cKellyDesign @p-netm @kahummer ?

Reply:
@moshthepitt you have a good point that this introduces an expectation that the component being typed is Reactful, which components don't necessarily need to be. The other types and utils in this package are all agnostic to types/code coming from other libraries or packages.

> on the other hand we are introducing React as a dependency

To me this alone is reason enough to move this `FlexComponent` typing into another package, perhaps a `ReactUtils`?

Reply:
> To me this alone is reason enough to move this `FlexComponent` typing into another package, perhaps a `ReactUtils`?

I have opended a PR for this https://github.com/onaio/js-tools/pull/147